Basic Mathematical Operations/C++Sol.cpp

```diff
@@ -0,0 +1,121 @@
+//This program consists of functions that are used to perform various basic mathematical operations such as addition , difference , multiplication
+// division , permutation and to check whether or not the two inputed values are divisble by one another , there is an integer varibale time that 
+// accepts the number of times the use wants to input the data and after the user inputs the value , all the functions are performed "time" times while
+//the user inputs the two values n times for each round of execution.
+
+
+#include<iostream>
+using namespace std;
+
+
+int number1 , number2 ;
+int sum=0 , prod , division , diff , permu1=1, permu2=1;
+
+void INPUT(); void SUM(); void DIFF();  void PROD(); void DIV(); void PERM(); void DIVIS();
+
+
+
+
+void INPUT()
+{
+	cout<<"\nenter two numbers"<<endl;
+	 cin>>number1>>number2;
+}
+void SUM()
+{
+	sum=number1+number2;
+	cout<<"Sum of "<<" "<<number1<<" "<<"and"<<" "<<number2<<" "<<"is :" <<sum<<endl;
+}
+
+void DIFF()
+{
+	 diff=number1-number2;
+	 cout<<"Absolute Difference of "<<" "<<number1<<" "<<"and"<<" "<<number2<<" "<<"is :" <<abs(diff)<<endl;
+}
+void PROD()
+{
+	prod=number1*number2;
+	 cout<<"Product of "<<" "<<number1<<" "<<"and"<<" "<<number2<<" "<<"is :" <<prod<<endl;
+}
+
+void DIV()
+{
```

Comment:
this code looks good but this is my suggestion
div_val = max(num1, num2)/min(num1, num2)
this take one operation more but more elegant.

Reply:
Yeah , that makes it a lot easier